main.go

```diff
@@ -113,7 +112,6 @@ func main() {
 	registerComponentOrExit(mgr, operatorsv1alpha1.AddToScheme)
 	registerComponentOrExit(mgr, argoapi.AddToScheme)
 	registerComponentOrExit(mgr, configv1.AddToScheme)
-	registerComponentOrExit(mgr, consolepluginv1.AddToScheme)
```

Comment:
@ciiay did you mean to update or remove this line? without it the console plugin cannot be deployed at all 
please add it back and update the reference